.pre-commit-config.yaml

```diff
@@ -66,6 +66,14 @@ repos:
     stages: [manual]
     pass_filenames: false
 
+  - id: check-unused-components
+    name: Check for unused components
+    entry: bash -c '$(which bazel) run //rs/ic_os/manifest_tool -- --repo-root $(git rev-parse --show-toplevel) check-unused-components'
```

Comment:
I'm a little paranoid about this check being in pre-commit. The check itself takes 10 seconds, but that's almost entirely bazel overhead (it takes less than a second when run with cargo).

I don't enforce pre-commit checks on every one of my commits, but if any others do, I'm not sure how much this will slow things down.

Reply:
> Would it make sense to run this on CI only?

I think so. Forgive my ignorance, but what would be the best way to plug this test into CI?